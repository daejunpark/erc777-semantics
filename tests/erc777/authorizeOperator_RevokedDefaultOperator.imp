test.preamble();

// Testing authorizeOperator(Operator) when Operator is a previously revoked default operator. 
// A token holder MUST be allowed to re-authorize a previously revoked default operator. 

test.revokeOperator(8);
test.authorizeOperator(8);
