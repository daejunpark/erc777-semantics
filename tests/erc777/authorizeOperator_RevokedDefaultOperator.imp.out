7 is revoking operator 8
7 is authorizing 8 as an operator
